ddtrace/profiling/_periodic.py

```diff
@@ -18,7 +19,14 @@ class PeriodicThread(threading.Thread):
 
     _ddtrace_profiling_ignore = True
 
-    def __init__(self, interval, target, name=None, on_shutdown=None):
+    def __init__(
+        self,
+        interval,  # type: float
+        target,  # type: typing.Callable[[], typing.Any]
+        name=None,  # type: typing.Optional[str]
+        on_shutdown=None,  # type: typing.Callable[[], typing.Any]
```

Comment:
curiosity question:

this format vs `#type: (float, typing.Callable[[], typing.Any], ...) -> None`

Reply:
really the same, it's just easier to manage and avoid changing a single line and messing with e.g. `git blame` 😆 
but both are fine imho.